Dockerfile

```diff
@@ -31,8 +32,11 @@ RUN set -x && \
     cp /root/ca/cacert.pem /usr/share/nginx/html/cacert.pem && \
     cp /root/ca/client.ansible.http.tests-cert.pem /usr/share/nginx/html/client.pem && \
     cp /root/ca/private/client.ansible.http.tests-key.pem /usr/share/nginx/html/client.key && \
-    pip install -c /root/constraints.txt gunicorn httpbin && \
-    apk del openssl-dev python2-dev libffi-dev gcc libstdc++ make musl-dev
+    chmod 644 /usr/share/nginx/html/* && \
+    pip install --no-cache-dir --no-compile -c /root/constraints.txt gunicorn httpbin && \
```

Comment:
Looks like `py3-pip` provides a `pip`, but maybe we should use `pip3` to be specific?

Reply:
That's fair, will update